require("./main").default().then(num => num && console.error("エラー発生"))